test/bdd/context.go

```diff
@@ -23,15 +25,19 @@ type Context struct {
 
 // NewContext create new Context
 func NewContext() (*Context, error) {
-	instance := Context{DIDExchangeClients: make(map[string]*didexchange.Client),
-		Invitations: make(map[string]*didexchange2.Invitation), PostStatesFlag: make(map[string]map[string]chan bool),
-		ConnectionID: make(map[string]string), AgentCtx: make(map[string]*context.Provider)}
+	instance := Context{
+		DIDExchangeClients: make(map[string]*didexchange.Client),
+		Invitations:        make(map[string]*didexchange2.Invitation),
```

Comment:
Suggest renaming the import aliases - it's hard to tell from the code what is coming from where.

Reply:
Also I don't understand why we are even importing from the protocol in a BDD test. It is supposed to be completely wrapped by DID exchange client.

@fqutishat @rolsonquadras @sandrask

Reply:
Issue https://github.com/hyperledger/aries-framework-go/issues/530 to fix this.